src/frontend/komponenter/Fagsak/Vilkårsvurdering/Vilkårsvurdering.tsx

```diff
@@ -75,15 +71,27 @@ const Vilkårsvurdering: React.FunctionComponent<IProps> = ({ fagsak, åpenBehan
         >
             <VilkårsvurderingSkjema visFeilmeldinger={visFeilmeldinger} />
 
-            {hentVilkårMedFeil().length > 0 && visFeilmeldinger && (
-                <Feiloppsummering
-                    tittel={'For å gå videre må du rette opp følgende:'}
-                    feil={hentVilkårMedFeil().map((vilkårResultat: IVilkårResultat) => ({
-                        feilmelding: `Vilkåret mangler resultat`,
-                        skjemaelementId: vilkårFeilmeldingId(vilkårResultat),
-                    }))}
-                />
-            )}
+            {(hentVilkårMedFeil().length > 0 || hentAndreVurderingerMedFeil().length > 0) &&
+                visFeilmeldinger && (
+                    <Feiloppsummering
+                        tittel={'For å gå videre må du rette opp følgende:'}
+                        feil={hentVilkårMedFeil()
+                            .map((vilkårResultat: IVilkårResultat) => ({
+                                feilmelding: `Vilkåret mangler resultat`,
+                                skjemaelementId: vilkårFeilmeldingId(vilkårResultat),
+                            }))
+                            .concat(
+                                hentAndreVurderingerMedFeil().map(
+                                    (annenVurdering: IAnnenVurdering) => ({
+                                        feilmelding: `Annen vurdering mangler resultat`,
+                                        skjemaelementId: annenVurderingFeilmeldingId(
+                                            annenVurdering
+                                        ),
+                                    })
+                                )
+                            )}
```

Comment:
```suggestion
                        feil={
                        [...hentVilkårMedFeil()
                            .map((vilkårResultat: IVilkårResultat) => ({
                                feilmelding: `Vilkåret mangler resultat`,
                                skjemaelementId: vilkårFeilmeldingId(vilkårResultat),
                            })),
                            ...hentAndreVurderingerMedFeil().map(
                                    (annenVurdering: IAnnenVurdering) => ({
                                        feilmelding: `Annen vurdering mangler resultat`,
                                        skjemaelementId: annenVurderingFeilmeldingId(
                                            annenVurdering
                                        ),
                                    })
                                )]}
```

Bedre å slippe å mutere på listene og bare bygge en ny liste her :) Er det riktig at man ønsker alle feil på vilkår først også andre vurderinger? Burde de kanskje vært gruppert per person nå som det potensielt kan bli flere "forskjellige" feil?